File: kotlinames-common/src/main/java/com/nagopy/android/kotlinames/property/nullable/KNullableFloatPropertyName.java
package com.nagopy.android.kotlinames.property.nullable;

import com.nagopy.android.kotlinames.KNullablePropertyName;
import com.nagopy.android.kotlinames.KSortablePropertyName;

public class KNullableFloatPropertyName implements KNullablePropertyName<Float>, KSortablePropertyName<Float> {

    private final String name;

    public KNullableFloatPropertyName(String name) {
        this.name = name;
    }

    @Override
    public String name() {
        return name;
    }
}

File: kotlinames/src/main/kotlin/com/nagopy/android/kotlinames/property/nullable/KNullableFloatPropertyName.kt
package com.nagopy.android.kotlinames.property.nullable

import com.nagopy.android.kotlinames.KNullablePropertyName
import com.nagopy.android.kotlinames.KSortablePropertyName

class KNullableFloatPropertyName(val name: String) : KNullablePropertyName<Float>, KSortablePropertyName<Float> {

    override fun name(): String = name

}
